src/v2/Apps/Partner/Components/NavigationTabs/index.tsx

```diff
@@ -7,13 +7,16 @@ import { RouteTab, RouteTabs } from "v2/Components/RouteTabs"
 import { StickyContainer } from "v2/Components/StickyContainer"
 import { NavigationTabs_partner } from "v2/__generated__/NavigationTabs_partner.graphql"
 
+// TODO: Update value in component heigth changed
+export const PARTHER_NAV_BAR_HEIGHT = 78
```

Comment:
non-blocking: but we could write some generic hook/context to get/set dom offset heights with refs 🤷

Reply:
👍 I'm gonna merge this in but @kiryl-zubarau feel free to create a ticket and follow up on this one!